libs/community/langchain_community/vectorstores/bigquery_vector_search.py

```diff
@@ -0,0 +1,827 @@
+"""Vector Store in Google Cloud BigQuery."""
+from __future__ import annotations
+
+import asyncio
+import json
+import logging
+import sys
+import uuid
+from datetime import datetime
+from functools import partial
+from threading import Lock, Thread
+from typing import Any, Callable, Dict, List, Optional, Tuple, Type
+
+import numpy as np
+from langchain.docstore.document import Document
```

Comment:
`community` should depend on `langchain_core` not on `langchain`

@baskaryan please, correct me if I'm wrong

Reply:
Done.